cpu/cortexm_common/Makefile.include

```diff
@@ -1,2 +1,6 @@
 # include module specific includes
 export INCLUDES += -I$(RIOTCPU)/cortexm_common/include
+
+all-debug: export CFLAGS += -g
```

Comment:
On second thought: it probably makes more sense to move the all-debug dependency declaration to some global location and purge it here and from the respective location in native.